lib/active_model/serializer.rb

```diff
@@ -21,7 +21,7 @@ class Serializer
     include Caching
     include Links
     include Type
-    require 'active_model/serializer/adapter'
+    require 'active_model_serializers/adapter'
```

Comment:
I think it's good. It has some extra spacing in one of the methods.

Maybe call it `configured_adapter`?